description/src/lib.rs

```diff
@@ -122,7 +181,7 @@ mod tests {
         let (type_id, type_registry) = make_type::<Human>();
 
         assert_eq!(
-            type_description(type_id, &type_registry).unwrap(),
+            type_description(type_id, &type_registry, true).unwrap(),
```

Comment:
Seeing the `Human { .. } ` bit below, and seeing variants like `Foo { .. }` above, gives me another nudge towards a "struct" prefix for clarity, so here we'd have `struct Human {  .. }`. Whatcha think?

Reply:
Hmm, whatever you think fits best. I had explicit `struct` and `enum` prefixes in the web app before and found it littered the entire code a bit, that is why I removed them. But it would be more consistent with Rust syntax.

Reply:
I changed it such that structs now have a `struct` prefix to make it more consistent.